packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1140,6 +1142,10 @@ class CupertinoDialogAction extends StatelessWidget {
     TextStyle style = _kCupertinoDialogActionStyle;
     style = style.merge(textStyle);
 
+    if (isDefaultAction) {
+      style = style.copyWith(fontWeight: FontWeight.bold);
```

Comment:
This isn't right. The font weight of an iOS preferred action is w600/semi-bold according to https://developer.apple.com/design/resources. Bold is w700.

Reply:
@xster Thanks! Didn't noticed that.

<img width="651" alt="Screenshot 2019-05-03 at 11 43 58" src="https://user-images.githubusercontent.com/6663880/57130232-21712d80-6d99-11e9-8e60-58a8fd61de95.png">